### DISCUSSION TITLE
Hide Kratos indices in Hunt [import install]

### DISCUSSION THREAD

**Discussion by stummasino-matc** [Security-Onion-Solutions/securityonion#9091] [General] [answered]
Is there a good way to automatically hide the kratos events from showing up in Hunt/Kibana. I am using a import only install and the first step I take each time is to filter out all of the kratos events. Seems like there should be a way to set it so those are filtered automatically (The kratos events were not showing up in earlier versions of the import install). I did try to find out how to do it with no luck. If someone can point me to a doc that explains it it would great :)

**Comment by dougburks** [comment, marked as answer]
When importing data using so-import-pcap or so-import-evtx, both of these utilities provide a hyperlink that will show you just the data from the import and nothing from Kratos or anything else:
https://docs.securityonion.net/en/2.3/so-import-pcap.html
https://docs.securityonion.net/en/2.3/so-import-evtx.html

**Reply by stummasino-matc** [reply]
Ah ok, Thanks Doug
